setup.sh

```diff
@@ -8,19 +8,15 @@ echo 'Installing macOS packages...'
 brew update
 brew install node
 brew install mongodb
-brew install grunt
```

Comment:
Actually I don't think we even need to install Grunt in the install script. [Grunt-cli basically puts the `grunt` command in your path and uses the local install of Grunt](https://www.npmjs.com/package/grunt-cli). Both yarn and npm consider programs from `node_modules/.bin`, where `grunt` would be. Since we're calling Grunt from a script in package.json, it _should_ work without needing to install `grunt-cli` globally.

ince we're calling Grunt from an npm script,